_includes/tutorials/anomaly-detection/ksql/markup/dev/print-accounts-to-monitor.adoc

```diff
@@ -0,0 +1,17 @@
+The ksqlDB table and thus Kafka topic contain a list of accounts against which more than three suspicious transactions have taken place in a 24-hour window.
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/anomaly-detection/ksql/code/tutorial-steps/dev/print-accounts-to-monitor.sql %}</code></pre>
++++++
+
+The output should look like the following:
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/anomaly-detection/ksql/code/tutorial-steps/dev/print-accounts-to-monitor.log %}</code></pre>
++++++
+
+Note that if you were to alter the `LIMIT` of results to something greater than 1, you would not see any other accounts flagged even though Victor von Frankenstein had a transaction that was flagged as suspicious. If you decided to rerun the query with a new limit, use `CTRL+D` to terminate the query. +
+
+
+This Kafka topic can be used to drive monitoring and alerting applications that could take action such as placing a hold on the account, notifying the card holder, etc. +
```

Comment:
1. "This Kafka topic" -- which Kafka topic?
2. Does the _topic_ drive, or maybe the _events_ drive?  (a la "event-driven" language)

Reply:
I altered this statement to use more event driven language per your suggestion.